### PR TITLE
Fix load from disk issue.

### PR DESCRIPTION
**Pull request by themisterholliday**
They keys were saving but slightly off every time they were being checked against each other

### CODE REVIEW

**Review by justinctlam — APPROVED** [review]

**Comment by justinctlam** [comment]
Now that we don't pull from the network every time, can you double check that the upvote/downvote is still respected? I don't think those are saved on Disk.

**Comment by themisterholliday** [comment]
Yea the upvote/downvote won't be respected in the next life cycle. I'll add a solution in this PR.

File: SEDaily-IOS/PodcastDataSource.swift [hunk 1]
```diff
@@ -107,24 +107,36 @@ class PodcastDataSource: DataSource {
...
+        do {
+            try Disk.save(items, to: .caches, as: DiskKeys.PodcastFolder.folderPath)
+        } catch let error {
+            log.error(error.localizedDescription)
+        }
     }
 
-    func clean() {
-        try? Disk.remove(DiskKeys.PodcastFolder.rawValue, from: .caches)
+    static func clean() {
```
Comment (justinctlam) [inline SEDaily-IOS/PodcastDataSource.swift:131]: Why is clean a static function? If you think about it, all of the functions in this class can be static as there's no internal state. Should we consider making all these functions static?

Comment (themisterholliday) [reply SEDaily-IOS/PodcastDataSource.swift:131]: Agreed. The datasource can be static. I'll make that change

**Pull request merged**